usr/src/test/test-runner/cmd/run.py

```diff
@@ -144,13 +145,16 @@ class Cmd(object):
     def __init__(self, pathname, outputdir=None, timeout=None, user=None):
         self.pathname = pathname
         self.outputdir = outputdir or 'BASEDIR'
-        self.timeout = timeout or 60
+        self.timeout = timeout
         self.user = user or ''
         self.killed = False
         self.result = Result()
 
+        if self.timeout == None:
+            self.timeout = 60
+
```

Comment:
if self.timeout is not an int, then you'll get ValueError thrown later at line 236 or 326
it is easy to check here first to ensure it is always convertible to int, something like:
```
try:
    self.timeout = int(self.timeout)
except ValueError as exc:
    # complain
    self.timeout = 60
```
then you'll be able to remove the `int(self.timeout)` later